README.md

```diff
@@ -3,7 +3,7 @@ cloudify-aws-plugin
 
 A Cloudify Plugin that provisions resources in Amazon Web Services
 
-* Master Branch [![Build Status](https://travis-ci.org/cloudify-cosmo/cloudify-aws-plugin.svg?branch=master)](https://travis-ci.org/cloudify-cosmo/cloudify-aws-plugin)
+* CFY-1892 Branch [![Build Status](https://travis-ci.org/cloudify-cosmo/cloudify-aws-plugin.svg?branch=CFY-1892)](https://travis-ci.org/cloudify-cosmo/cloudify-aws-plugin)
```

Comment:
because master is not passing build tests until i merge the fixes in CFY-1879. And then you get this ugly "failed" sticker on the readme page for this branch